content/millennium/r4/request-and-response/communication.md

```diff
@@ -61,13 +61,13 @@ _Implementation Notes_
  Name         | Required?                                       | Type          | Description
 --------------|-------------------------------------------------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------
  `_id`        | This or `category` or `recipient` or `received` | [`token`]     | The logical resource id associated with the resource. Example: `C-1123_0`
- `category`   | This or `_id` or `recipient` or `received`      | [`token`]     | A token for a [`CodeableConcept`] that points to the CodeableConcept used for Communication.category. Example: `category=http://terminology.hl7.org/CodeSystem/communication-category|notification` or `category=http://terminology.hl7.org/CodeSystem/communication-category|reminder`
- `recipient`  | This or `_id` or `category`  or `received`      | [`reference`] | The recipient of the Communication. Example: `recipient=Practitioner/3456783` or `recipient:Group=8765556`
- `received`   | This or `_id` or `category`  or `recipient`     | [`date`]      | Date range into which the communication falls. Example: `received=le2017-02-01T10:30:00Z` or `received=ge2020-03-16T21:31:35.458Z&received=le2020-07-16T21:31:35.457Z"`
+ `category`   | This or `_id` or `recipient` or `received`      | [`token`]     | A token for a [`CodeableConcept`] that points to the CodeableConcept used for the communication.category. Example: `http://terminology.hl7.org/CodeSystem/communication-category|notification`
```

Comment:
This shouldnt be changed. It's a field path.

Reply:
Reverted: https://github.com/cerner/fhir.cerner.com/pull/440/commits/cf002fdb54739e842215998ff3ed7e977152dc2b and updated screenshot